pkg/nsx/services/ippool/builder.go

```diff
@@ -0,0 +1,95 @@
+package ippool
+
+import (
+	"strings"
+
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha2"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+	"github.com/vmware-tanzu/nsx-operator/pkg/util"
+)
+
+var (
+	Int64  = common.Int64
+	String = common.String
+)
+
+func (service *IPPoolService) BuildIPPool(IPPool *v1alpha2.IPPool) (*model.IpAddressPool, []*model.IpAddressPoolBlockSubnet) {
```

Comment:
Can we only return one variable ippool? The concern is that we'd better keep a unify common style. My thinking is that we'd better had a function template likes as follow:
```
func (service *<CRD name>Service) Build<CRD name>(<CRD name> *<api version>.<CRD name>) (*model.<NSX resource type>) {
    return &model.<NSX resource type>{
        // TODO: add properties here
    }
}
```
What's your opinion?

Reply:
Then security policy also requires change, much effort needed, perhaps.